go/src/koding/klientctl/main.go

```diff
@@ -49,6 +49,11 @@ var sudoRequiredFor = []string{
 // cli rewrite.
 var log logging.Logger
```

Comment:
could you create this logger with debug parameter is if `KD_DEBUG` exists? and remove https://github.com/koding/koding/pull/9182/files#diff-9d1c92b05bf71fa597eb81b434721addR62
```
+	if debug {
 +		log.SetLevel(logging.DEBUG)
 +		k.SetLogLevel(kite.DEBUG)
 +	}
```